home.html

```diff
@@ -12,12 +12,12 @@
 <body class="body">
     <div class="header">
         <div class="header__logoIcon">
-            <img src="/Assets/images/home/company_logo.svg" alt="Company Logo">
+            <img src="/Assets/images/home logo/company_logo.svg" alt="Company Logo">
             <h1 class="header__logoTitle">Pet Care</h1>
         </div>
 
         <div class="sandwich_menu">
-            <img src="/Assets/images/home/cat-logo.svg" alt="">
+            <img src="/Assets/images/home logo/cat-logo.svg" alt="">

```

Comment:
These changes are not really related to the issue and PR, right?

Reply:
They are not indeed, i should have made another branch for that change